### ISSUE TITLE
R-Instat download flagged by windows defender as malicious?

### ISSUE THREAD

**Issue by shadrackkibet** [IDEMSInternational/R-Instat#7627]
What can be done about the following? Windows defender flags R-Instat download as potentially malicious. A possible reason could be because the website isn't secured therefore it can be subject to a MITM.

![image](https://user-images.githubusercontent.com/26301973/179351292-7862a13e-9038-443a-b8e9-246b8adb044a.png)
![image](https://user-images.githubusercontent.com/26301973/179351381-b7311c10-73e9-4f07-90a9-fef42ee72e43.png)
![image](https://user-images.githubusercontent.com/26301973/179351721-2956e148-d03e-41b4-9332-bfc43114b54d.png)

**Comment by shadrackkibet** [comment]
Aha! I guess that's because it is missing the digital signature!
![image](https://user-images.githubusercontent.com/26301973/179351945-6eabe278-fc25-4355-8680-0f69f51f33ca.png)

**Comment by rdstern** [comment]
@shadrackkibet you are right to be concerned.  We have the digital signature and should have it in place by the end of next week.
